partials/accept-terms.html

```diff
@@ -20,7 +20,7 @@
 		<p class="terms-corp-signup">myFT topics may be selected on your behalf by your company administrator or FT representative for you to follow. You can unfollow these topics or unsubscribe from the myFT digest through the Contact preferences section on myFT.</p>
 
 			{{#if isTrial}}
-			<p class="terms-corp-signup">This trial is to demonstrate the value of a group subscription and we'll contact you during your trial.</p>
+			<p class="terms-corp-signup">This trial is to demonstrate the value of a group subscription and we‘ll contact you during your trial.</p>
```

Comment:
This is pedantry at its finest, but shouldn't this be the following?

```suggestion
			<p class="terms-corp-signup">This trial is to demonstrate the value of a group subscription and we’ll contact you during your trial.</p>
```